### PR TITLE
Oberth stuff, cloth and armor pack by Armolitskiy

### PR DESCRIPTION
**Pull request by Tea-Primoris**
## About The Pull Request
This PR adds German themed outfit and a crate. Crate with full outfit will spawn randomly as and individual parts of outfit. Sprites by Armolitskiy (Armolitskiy#4866).

## Sprites
<details><summary>Uniform and boots</summary>

image image image</details>

<details><summary>Outfit without gas mask</summary>

image image image
</details>

<details><summary>Full outfit</summary>

image image image
</details>

<details><summary>Crate</summary>

image image
</details>

## Changelog
:cl:
add: New German themed outfit
imageadd: New sprites from Armolitskiy
tweak: Crate with outfit and individual parts of outfit will spawn randomly
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/mercenary.dm [hunk 1]
```diff
@@ -27,4 +27,11 @@
...
-	icon_state = "serbiansuit_black"
\ No newline at end of file
+	icon_state = "serbiansuit_black"
+
+/obj/item/clothing/under/germansuit
+	name = "brown Battle Dress Uniform"
+	desc = "A tough, wear-resistant battle dress uniform in brown colors. Typically worn underneath armour"
+	icon_state = "germansuit_brown"
+	item_state = "germansuit_brown"
+	has_sensor = 0
```
Comment (Pariah919) [inline code/modules/clothing/under/mercenary.dm:37]: add another line.

File: code/modules/clothing/head/misc.dm [hunk 2]
```diff
@@ -247,3 +247,8 @@
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|BLOCKHAIR
 	body_parts_covered = HEAD|FACE|EARS
 	siemens_coefficient = 1
+
+/obj/item/clothing/head/beret/german
```
Comment (SpadesSlick) [inline code/modules/clothing/head/misc.dm:251]: Beret needs at least some basic energy armor, 25 will be fine. Also you need to write about that in description.

File: code/modules/clothing/head/misc.dm [hunk 3]
```diff
@@ -247,3 +247,8 @@
...
+	name = "beret"
+	desc = "Brown beret with emblem. For those that are more inclined towards style than safety."
+	icon_state = "germanberet"
```
Comment (SpadesSlick) [inline code/modules/clothing/head/misc.dm:254]: Endlines.

File: code/modules/clothing/suits/armor.dm [hunk 4]
```diff
@@ -205,6 +205,19 @@
 
 			return PROJECTILE_CONTINUE // complete projectile permutation
 
+/obj/item/clothing/suit/armor/greatcoat/german_overcoat
+	name = "black german overcoat"
+	desc = "A black german overcoat with armored plates"
```
Comment (SpadesSlick) [inline code/modules/clothing/suits/armor.dm:210]: You need to mention that it made from special materials that negate energy damage better.

File: code/modules/clothing/suits/armor.dm [hunk 5]
```diff
@@ -205,6 +205,19 @@
 
 			return PROJECTILE_CONTINUE // complete projectile permutation
 
+/obj/item/clothing/suit/armor/greatcoat/german_overcoat
+	name = "black german overcoat"
```
Comment (SpadesSlick) [inline code/modules/clothing/suits/armor.dm:209]: Not "black german" but "Oberth Republic uniform overcoat". Also mention in description that this is from surplus.

File: code/modules/clothing/under/mercenary.dm [hunk 6]
```diff
@@ -27,4 +27,12 @@
 /obj/item/clothing/under/serbiansuit/black
 	name = "black Battle Dress Uniform"
 	desc = "A tough, wear-resistant battle dress uniform in urban colors. Typically worn underneath armour"
-	icon_state = "serbiansuit_black"
\ No newline at end of file
+	icon_state = "serbiansuit_black"
+
+/obj/item/clothing/under/germansuit
+	name = "brown Battle Dress Uniform"
```
Comment (SpadesSlick) [inline code/modules/clothing/under/mercenary.dm:33]: Republic should also be mentioned on every item.

**Comment by SpadesSlick** [comment]
Also wrong kind of crates are used, we need item based ones. I don't know why serbs got both of those, lol.

**Pull request merged**